packages/pacget-git/pacget-git.pacscript

```diff
@@ -0,0 +1,44 @@
+#!/bin/env bash
+#-*- mode: sh ;mode:lsp;-*-
+name="pacget-git"
+pkgname="pacget"
+pkgdir="/usr/src/pacstall/${name}"
+
+maintainer="Erik Lundstedt <erik@lundstedt.it>"
+url="https://gitlab.com/Erik.Lundstedt/pacget.git"
+
+license="Don't Be a Jerk"
```

Comment:
```suggestion
license="Don't Be a Jerk"
```
Not needed!

Reply:
booth the program itself and the pacscript have that licence and i prefer putting the licence-notice on there even if its not needed